### ISSUE TITLE
Multiple workspaces with the same workspace name

### ISSUE THREAD

**Issue by tdari** [Tauffer-Consulting/domino#230]
Currently, you can't create a workspace with the same name if it already exists. But through invitation, a user can be in multiple workspaces with the same name. I think it is something that we don't want to happen.

![image](https://github.com/Tauffer-Consulting/domino/assets/50070971/bea40f23-09cc-406e-aa9d-c616240ad0ea)

I propose to update the workspace model to have a field called user_name which derives from the user's email field.
When the user creates a new workspace we add user_name to the beginning of the workspace name. Therefore, a user can no longer have multiple workspaces because all workspaces are prefixed with the workspace's owner's name. In fact, it is what cloud-based Git repository providers are using.

We can also update the UI to look like this or we can keep the current design and just save with the prefix:

![image](https://github.com/Tauffer-Consulting/domino/assets/50070971/6e019b53-3d2b-4799-9871-5704c2d2dae0)

Here is how it will look like after the change,

![image](https://github.com/Tauffer-Consulting/domino/assets/50070971/357cf059-dc79-4984-98f0-3173204075fd)

We also need to update the default workspace creation configuration accordingly.

**Comment by tdari** [comment]
@vinicvaz Yes permissions can be updated as well like you said. Email can be used instead of a username as you said but I could not imagine how it would look like on the UI. I think this workspace update needs to be devised carefully maybe it is better to fix current obvious bugs first what do you think?

**Comment by vinicvaz** [comment]
Agree with you, I'm planning to look again to the database design this week and I will think about workspaces permissions.


About other bugs, I found one yesterday you can look if you have availability, is this one: https://github.com/Tauffer-Consulting/domino/issues/229
